### ISSUE TITLE
 generate_voltera of stochastic_process.voltera and RoughBergomiStock of instruments.primary are slightly different

### ISSUE THREAD

**Issue by jay-ylee** [anh-tong/sigformer#1]
Hello, thank you for the code you implemented with jax. I am posting an issue because there seems to be a slight difference between the generate-volterra of the stochastic process module and the volterra (in bergomi) you implemented in the class within the primary. I think we should pad X after the 184th line of the simple method in the RoughBergomi Stock class(in the generate-volterra function in the stochastic process, we pad it before convolution, line 51). If there's anything I missed, please let me know.

**Comment by jay-ylee** [comment]
Oh, I see again that padding is wrong in generate_volterra in the stochastic process.
